packages/runner/webpack.config.ts

```diff
@@ -84,16 +84,47 @@ mainConfig.resolve = {
 }
 
 // @ts-ignore
-const injectionConfig: webpack.Configuration = {
+const crossOriginConfig: webpack.Configuration = {
+  mode: 'development',
```

Comment:
why not `mode: 'production'`? i'm unsure why this would differ from the other configs

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/21137/commits/ebd40c7fa5978ff9656e6a90112ecd31af569a2d